ironfish-cli/src/multisigBroker/server.ts

```diff
@@ -186,7 +186,8 @@ export class MultisigServer {
       )
 
       if (parseError) {
-        this.sendErrorMessage(client, 0, `Error parsing message`)
+        client.close(parseError)
```

Comment:
What do you think about adding a debug log here for potential future debugging?

Reply:
I think that's a great idea!